.gitlab-ci.yml

```diff
@@ -120,6 +120,8 @@ test:behat:
   after_script:
   - cd ${BUILD_DIR}
   - make browser_driver_stop
+  - mv features/*.png web/
```

Comment:
It should be added to separate dir and dxposed as artefacts to allow view in gitlab

Reply:
I bet the screen shots dir configurable